packages/autocertifier-client/src/data/CertifiedSubdomain.ts

```diff
@@ -0,0 +1,8 @@
+import { Certificate } from './Certificate'
+
+export interface CertifiedSubdomain {
+    subdomain: string
```

Comment:
Is this redundant field as the subsomain in fqdn must always match the subdomain?